src/CalcViewModel/Common/CalculatorDisplay.cpp

```diff
@@ -49,6 +49,18 @@ void CalculatorDisplay::SetParenDisplayText(_In_ const std::wstring& parenthesis
     }
 }
 
+void CalculatorDisplay::OnNoParenAdded()
+{
+    if (m_callbackReference != nullptr)
+    {
+        auto calcVM = m_callbackReference.Resolve<ViewModel::StandardCalculatorViewModel>();
+        if (calcVM)
```

Comment:
Even better is:
```C++
if (auto calcVM = ...)
{

}
```
This style has the exact same effect, except the `calcVM` variable is scoped to the body of the `if` block and does not exist outside, which is the best reflection of intent here.

Reply:
I fixed this throughout the file.

---
In reply to: [264936066](https://github.com/Microsoft/calculator/pull/284#discussion_r264936066) [](ancestors = 264936066)

Reply:
Thanks!